Sources_v3/StreamChat/APIClient/Endpoints/Payloads/MessagePayloads.swift

```diff
@@ -25,12 +25,16 @@ enum MessagePayloadsCodingKeys: String, CodingKey {
     case ownReactions = "own_reactions"
     case reactionScores = "reaction_scores"
     case isSilent = "silent"
+    case channel
     //        case i18n
 }
 
-/// An object describing the wrapped incoming message JSON payload.
-struct WrappedMessagePayload<ExtraData: ExtraDataTypes>: Decodable {
-    let message: MessagePayload<ExtraData>
```

Comment:
I liked `WrappedMessage` more than `MessagePayload.Box` :( Any particular reason to change it?

Reply:
@b-onc I didn't like the name collision with optional wrapped type + I kinda like having this a nested type. Happy to hear your suggestions regarding the name.

Reply:
I'm a `Box` guy

Reply:
On other hand from usage point of view I would expect `ed` suffix:
```
let endpoint: Endpoint<MessagePayload<DefaultExtraData>.Boxed>
let endpoint: Endpoint<MessagePayload<DefaultExtraData>.Wrapped>
```

Reply:
I like the `Boxed` idea. Fixed in 2da2168